deploy/crds/performance.openshift.io_performanceprofiles_crd.yaml

```diff
@@ -54,6 +54,11 @@ spec:
                   description: Isolated defines a set of CPUs that will be used to
                     give to application threads the most execution time possible,
                     which means removing as many extraneous tasks off a CPU as possible.
+                    It important to notice, that the CPU manager can choose any CPU
+                    to run the workload, except reserved CPUs, so to guarantee that
```

Comment:
change to :  to run the workload except reserved CPUs. In order to guarantee...

Reply:
done